pkg/controllers/service/service_lb_controller.go

```diff
@@ -55,24 +61,31 @@ func (r *ServiceLbReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 	}()
 
 	if err := r.Client.Get(ctx, req.NamespacedName, service); err != nil {
+		if client.IgnoreNotFound(err) == nil {
+			log.Info("Not found LB service", "req", req.NamespacedName)
+			return ResultNormal, client.IgnoreNotFound(err)
```

Comment:
Do we need yo requeue for this not found case?

Reply:
not found means it's been deleted. no need to requeue it